### DISCUSSION TITLE
i Ask for complied ver.

### DISCUSSION THREAD

**Discussion by eltabry** [OpenSalamander/salamander#46] [General]
I use the program from the first version until the latest version, and I wish there was an executable version of the program because I have no experience in programming.

**Comment by melloware** [comment]
I think they are working on automating the build with GitHub Actions to produce the compiled binary.

**Comment by eltabry** [comment]
stell waiting for complied Ver.

**Comment by Akrotkiv** [comment]
> I use the program from the first version until the latest version, and I wish there was an executable version of the program because I have no experience in programming.

Me the same. I tried many file managers parallel with Salamander, but always ended back to Salam. Even when the problems with unicode came. I bought a license to support it... 
Now it is very sad, that unicode support is by the hand, but not reachable.

Please give us  (non-programmers) a little gift to Christmas - an compiled version to test and try ;)

**Reply by OFreddy** [reply]
> I use the program from the first version until the latest version, and I wish there was an executable version of the program because I have no experience in programming.

Me also. I was very happy to find the Repository on GitHub. 

I started compiling but unfortunately I am facing false positives on some binaries with my virus protection.

Anyhow you can try the compiled version in my fork as long as there is no other solution.
[https://github.com/OFreddy/salamander/releases/tag/v5.00](https://github.com/OFreddy/salamander/releases/tag/v5.00)

This version is inconspicuous on Virustotal.com.

![2024-12-04 13_58_38-VirusTotal - File - d3c4ff9439136ec495c8f21e23ec037a668a17029712c2550401cddcedbc](https://github.com/user-attachments/assets/ed5de852-79a3-4d9c-9d41-c75701de55fd)

You can execute the program by selecting salamand.exe in Release_x64 folder.

**Reply by Akrotkiv** [reply]
Great! Thank you.
Is there any problem by using it? Is it "stable"?
I will try it for a while.
There was just problem by "installing" RARplugin.
I don't know how the unicode problem is handeled.
At first sight it works (copy utf-8 named files). But some weired appearing: The same file (copied from right to left panel) doesnt appear correctly.
![2024-12-04_173454](https://github.com/user-attachments/assets/9d99f3ec-32ff-457e-bca6-dda9b2be900b)
and some "strange" symbol in file size
![openmander](https://github.com/user-attachments/assets/34e2ec1d-61a0-4f1a-8324-b1730be4ac59)
But this is OFF-topic here ;)

**Reply by OFreddy** [reply]
> Is there any problem by using it? Is it "stable"?

It is very stable considering my requirements. I really enjoy the advantages over working with Windows Explorer.
Unicode support is not an issue for me. As far as I can remember, support for Unicode file names has always been a problem. There are various discussions about it, also on the Altap homepage. I think the scope of this tool is different, just perfect for my stuff.
If you have problems with individual files, simply use Windows Explorer to process them.

**Reply by melloware** [reply]
When I try and download your zip Microsoft Defender says there is a virus in it.  So I see what you are saying.
